tests/Controller/WizardControllerTest.php

```diff
@@ -95,71 +95,6 @@ public function testDisable($user) {
 		$this->assertSame(Http::STATUS_OK, $response->getStatus());
 	}
 
-
-	public function testShowUser() {
-		$this->config->expects($this->once())
-			->method('getAppValue')
-			->with(Application::APP_ID, 'slides', 'video,values,apps,clients,final')
-			->willReturnArgument(2);
-		$controller = $this->getController();
-		$this->config->expects($this->exactly(5))
-			->method('getSystemValue')
-			->willReturnMap([
-				['customclient_desktop', 'https://nextcloud.com/install/#install-clients', 'https://nextcloud.com/install/#install-clients'],
-				['customclient_android', 'https://play.google.com/store/apps/details?id=com.nextcloud.client', 'https://nextcloud.com/install/#install-clients'],
-				['customclient_fdroid', 'https://f-droid.org/packages/com.nextcloud.client/', 'https://nextcloud.com/install/#install-clients'],
-				['customclient_ios', 'https://geo.itunes.apple.com/us/app/nextcloud/id1125420102?mt=8', 'https://nextcloud.com/install/#install-clients'],
-				['appstoreenabled', true, true]
```

Comment:
@marcoambrosini I fixed the tests. However we should probably re-introduce this so that customers can adjust the links. Should we do this still in this PR or rather in a follow-up like the transitions?

Reply:
Put it into https://github.com/nextcloud/firstrunwizard/issues/985